### PR TITLE
Added UnRAID Docker install, formatting fixes

### PR DESCRIPTION
**Pull request by suckerface**

### CODE REVIEW

**Comment by suckerface** [comment]
To clarify on the formatting fixes, the numbered lists were not rendering properly due to the extra line breaks. Escaped the `.` for the Docker install instructions, and removed line breaks in UnRAID docker install instructions.

File: docs/user-guide/install.md [hunk 1]
```diff
@@ -24,19 +24,19 @@ VAAPI hardware-accelerated transcoding: `jasongdove/ersatztv:develop-vaapi`
 
 ### Docker
 
-1. Download the latest container image
```
Comment (suckerface) [inline docs/user-guide/install.md:27]: #100 
These are the line breaks that are breaking the ordered list. By escaping `.` for steps 2 and 3, the step numbers render properly.

File: docs/user-guide/install.md [hunk 2]
```diff
@@ -51,6 +51,28 @@ docker run -d \
...
+
+```
+https://raw.githubusercontent.com/Squidly271/community.applications/master/plugins/community.applications.plg
+```  
+
+ ![Community Applications Install](../images/unraid-install-community-apps.png)  
+2. Click on the `Apps` tab in UnRAID, search for `ersatztv`, and click the `install` button.
+
+![Search Community Applications ErsatzTV](../images/unraid-docker-install-ersatz.png)
+3. Choose an appropriate branch (Use `vaapi` for video acceleration for AMD GPUs and Intel CPUs with QuickSync, use `nvidia` for video acceleration for nVidia GPUs)
```
Comment (suckerface) [inline docs/user-guide/install.md:64]: #100 
Removing the line break between the image link and the next steps renders the step numbers properly.

**Review by jasongdove — APPROVED** [review]
Looks good, thanks!

**Pull request merged**